README.md

```diff
@@ -4,4 +4,23 @@ This repository contains config files and scripts used for a CI workflow at Digi
 
 ## Structure
 
-There is a hierarchical folder structure that gives more specific config files in deeper folders. So you start in the most specific applicable folder and work your way up to use the config files.
\ No newline at end of file
+There is a hierarchical folder structure that gives more specific config files in deeper folders. So you start in the most specific applicable folder and work your way up to use the config files.
+
+## Extend
+
+It is possible to extend the config files. In order to do so:
```

Comment:
Maybe a separate section for .eslint?
If there are other files, we can just add sections.